buf.gen.yaml

```diff
@@ -0,0 +1,6 @@
+version: v1
+managed:
+  enabled: true
+plugins:
+  - plugin: buf.build/protocolbuffers/php
```

Comment:
@spawnia Do you want to pin a specific protobuf version or always use the latest?

Reply:
Can we pin a major version to avoid incidental breaking changes but still allow for bugfixes?

Reply:
The plugin follows the protobuf version number (and there is a new "major" version every few months but I think they don't follow semver) but usually there are no breaking changes in generated PHP code. 
In the last 4/5 releases that I used they added some phpdoc to improve types

Here are the release pages:
https://buf.build/protocolbuffers/php
https://github.com/protocolbuffers/protobuf/releases